.travis.yml

```diff
@@ -28,13 +31,26 @@ install:
 
   # Install build dependencies.
   # See also `apt-cache showsrc awesome | grep -E '^(Version|Build-Depends)'`.
-  - sudo apt-get install -y libcairo2-dev xmlto asciidoc libpango1.0-dev gperf luadoc libxcb-xtest0-dev libxcb-icccm4-dev libxcb-randr0-dev libxcb-keysyms1-dev libxcb-xinerama0-dev libxcb-image0-dev libev-dev libimlib2-dev libdbus-1-dev libxdg-basedir-dev libstartup-notification0-dev imagemagick libxcb1-dev libxcb-shape0-dev libxcb-util0-dev libxcursor-dev libx11-xcb-dev libxcb-cursor-dev libxcb-xkb-dev libxkbcommon-dev libxkbcommon-x11-dev
+  - sudo apt-get install -y libcairo2-dev xmlto asciidoc libpango1.0-dev libxcb-xtest0-dev libxcb-icccm4-dev libxcb-randr0-dev libxcb-keysyms1-dev libxcb-xinerama0-dev libdbus-1-dev libxdg-basedir-dev libstartup-notification0-dev imagemagick libxcb1-dev libxcb-shape0-dev libxcb-util0-dev libx11-xcb-dev libxcb-cursor-dev libxcb-xkb-dev libxkbcommon-dev libxkbcommon-x11-dev
 
   # Deps for functional tests.
   - sudo apt-get install -y dbus-x11 xterm xdotool xterm xvfb rxvt-unicode
 
   # Install Lua (per env).
-  - sudo apt-get install -y lib${LUANAME}-dev ${LUANAME} ${INSTALL_PKGS}
+  # Note that Lua 5.3 is installed by hand, because not available in trusty
+  - |
```

Comment:
Apparently it does. I was "inspired by" (= stole it from) here: https://github.com/znc/znc/blob/2bcc5c51593fd63de5f0e2757bbfbff13451b06d/.travis.yml#L60-L65 The only downside is that I need that `|| exit 1` else Travis does not fail if a subcommand fails.

I just don't know why Travis didn't try another rebuild with my additionally pushed commit which "added" `/usr/bin/lua` (previously `build-utils/lgi-check.sh` was using lua5.1 which was pulled in by Luadoc, because I removed Luadoc this fails now). Any ideas?